integration-tests/virtual-threads/jms-virtual-threads/pom.xml

```diff
@@ -37,7 +37,7 @@
         <dependency>
             <groupId>io.smallrye.reactive</groupId>
             <artifactId>smallrye-reactive-messaging-jms</artifactId>
-            <version>4.12.0</version>
+            <version>4.15.0</version>
```

Comment:
Should we include this one in the BOM so that it's updated properly when we update SmallRye Reactive Messaging?

/cc @cescoffier @ozangunalp

Reply:
Or perhaps we could import the `io.smallrye.reactive:smallrye-reactive-messaging-bom` that manages these artifacts?